tests/slate/test_slate_infrastructure.py

```diff
@@ -66,15 +66,21 @@ def load(function_space):
 @pytest.fixture
 def boundary_load(function_space):
     f = Function(function_space)
-    f.interpolate(Expression("cos(x[1]*pi*2)"))
+    if function_space.mesh().cell_dimension == 1:
```

Comment:
Good catch. That was totally my bad. I wrote this test ages ago and I guess I just wasn't careful. Thanks for fixing this